pkgs/stdenv/linux/default.nix

```diff
@@ -546,6 +548,7 @@ in
         ++  [ /*propagated from .dev*/ linuxHeaders
             binutils gcc gcc.cc gcc.cc.lib gcc.expand-response-params
           ]
+          ++ lib.optionals (localSystem.libc == "musl") [ fortify-headers ]
           ++ lib.optionals (!localSystem.isx86 || localSystem.libc == "musl")
```

Comment:
Can we use ``isMusl`` here?

Reply:
I suspect not because `stdenv` is probably not even the _target_ stdenv here, rather the previous stage's stdenv.